README.md

```diff
@@ -3,14 +3,100 @@
 
 This is the Engine.IO Server Library for Java ported from the [JavaScript server](https://github.com/socketio/engine.io).
 
+See also: [Engine.IO-client Java](https://github.com/socketio/engine.io-client-java)
+
+## Installation
+This section will be updated when artifact is available on Maven Central.
+
+### Maven
+This section will be updated when artifact is available on Maven Central.
+
+### Gradle
+This section will be updated when artifact is available on Maven Central.
+
 #### Engine.IO Protocol 1.x suppport
 
 The current version of engine.io-java does not support protocol 1.x.
 
 ## Usage
 Usage is slightly different based on the server being used.
 
-TODO: Add usage for Jetty and Tomcat.
+Create a servlet to handle the HTTP requests as follows:
+```java
+public class EngineIoServlet extends HttpServlet {
+
+    private static final EngineIoServer ENGINE_IO_SERVER = new EngineIoServer();
+
+    public static EngineIoServer getEngineIoServer() {
+        return ENGINE_IO_SERVER;
+    }
+
+    @Override
+    protected void service(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        ENGINE_IO_SERVER.handleRequest(request, response);
+    }
+}
+```
+
+Listen for new connections as follows:
+```java
+EngineIoServer server = EngineIoServlet.getEngineIoServer();
+server.on("connection", new Emitter.Listener() {
+    @Override
+    public void call(Object... args) {
+        EngineIoSocket socket = (EngineIoSocket) args[0];
+        // Do something with socket
+    }
+});
+```
+
+Listen for packets as follows:
+```java
+EngineIoSocket socket;
```

Comment:
Where does the `socket` object come from?

Reply:
The socket object should come from the servlet or other place as defined by the user. I'll make it more clear.